### PR TITLE
add docker file

### PR DESCRIPTION
**Pull request by netanelbarel**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM node:10-alpine
```
Comment (ItayHoresh) [inline Dockerfile]: why 10?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    
+# Create app directory
+WORKDIR /usr/src/app
+
+# RUN npm ci --only=production
+
+# Bundle app source
+COPY . .
+
+EXPOSE 3000
```
Comment (ItayHoresh) [inline Dockerfile]: change to 8080

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+version: '3.7'
+
+services:
+
+  pipe-rt-manager:
+    container_name: pipe-rt-manager
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
```
Comment (ItayHoresh) [inline docker-compose.yaml]: What is it for?

**Comment by ItayHoresh** [comment]
We need to think how to merge this project to the pipert project

https://github.com/ItamarWilf/PipeRT/issues/41

File: docker-compose.yaml (hunk 3 above)
Comment (netanelbarel) [reply docker-compose.yaml]: easy to run locally 
docker-compose up -d  --build
instead of 
docker build .....
docker run ......

Comment (ItayHoresh) [reply docker-compose.yaml]: what is the args for?

File: Dockerfile [hunk 4]
```diff
@@ -1,4 +1,4 @@
-FROM node:10-alpine
+FROM node:alpine3.11
```
Comment (ItayHoresh) [inline Dockerfile]: @DorTruzman what is the recommended version of node? What is your version?

File: docker-compose.yaml [hunk 5]
```diff
@@ -13,4 +13,4 @@ services:
         no_proxy:
         socks_proxy:
     ports:
-      - '3000:3000'
\ No newline at end of file
+      - '8080:3000'
```
Comment (ItayHoresh) [inline docker-compose.yaml]: @DorTruzman Did you changed the port to 8080? If so, it supposed to be 8080:8080

Comment (DorTruzman) [reply docker-compose.yaml]: Changed to 8080

File: Dockerfile (hunk 4 above)
Comment (DorTruzman) [reply Dockerfile]: The newest version should be fine

**Pull request merged**